### PR TITLE
Added Test Formulas and Test formulas tutorial

### PR DESCRIPTION
**Pull request by jessica-tw**
- Added 'Test formula' to Formulas section; 
- Added new tutorial 'How to test formulas outputs' 
- Created a Contribute section to be moved to the menu.

### CODE REVIEW

File: content/pt-br/Fórmulas/testar formulas.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Testar formulas 
+weight: 51
+description: 'Nesta seção, você vai saber mais como testar fórmulas no Ritchie.'
+---
+
+---
+
+## **Testes Unit** 
+
+No contexto do Ritchie CLI, Unit tests é a forma de testar uma parte da implementação da sua fórmula.
```
Comment (GuillaumeFalourd) [inline content/pt-br/Fórmulas/testar formulas.md:11]: Podemos traduzir `Unit tests` para `testes unitários`

File: content/pt-br/Fórmulas/testar formulas.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+description: 'Nesta seção, você vai saber mais como testar fórmulas no Ritchie.'
+---
+
+---
+
+## **Testes Unit** 
+
+No contexto do Ritchie CLI, Unit tests é a forma de testar uma parte da implementação da sua fórmula. 
+
+Alguns templates de fórmula, como Java e Golang possuem classe de testes para escrever e rodar testes unit das fórmulas.
```
Comment (GuillaumeFalourd) [inline content/pt-br/Fórmulas/testar formulas.md:13]: Substituir `unit` por `unitário`

**Review by GuillaumeFalourd — CHANGES_REQUESTED** [review]

File: content/pt-br/Fórmulas/testar formulas.md (hunk 1 above)
Comment (jessica-tw) [reply content/pt-br/Fórmulas/testar formulas.md]: done! ;)

**Review by GuillaumeFalourd — APPROVED** [review]
👍🏼

**Pull request merged**